### PR TITLE
Add Sendcloud

### PR DESCRIPTION
**Pull request by AlexLamond**
Add sendcloud.com (account.sendcloud.com) to the 2FA list

### CODE REVIEW

File: entries/s/sendcloud.com.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "Example Site": {
+    "domain": "sendcloud.com",
+    "url": "https://account.sendcloud.com",
```
Comment (Carlgo11) [inline entries/s/sendcloud.com.json]: The main url of the service is https://www.sendcloud.com/ not the accounts subdomain.
```suggestion
```

File: entries/s/sendcloud.com.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "Example Site": {
```
Comment (Carlgo11) [inline entries/s/sendcloud.com.json]: This element specifies the name of the service. As such it should have the value `Sendcloud`
```suggestion
  "Sendcloud": {
```

File: entries/s/sendcloud.com.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "domain": "sendcloud.com",
+    "url": "https://account.sendcloud.com",
+    "img": "sendcloud.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.sendcloud.com/hc/en-gb/articles/4418777976212-Change-your-password-manage-your-Sendcloud-account",
```
Comment (Carlgo11) [inline entries/s/sendcloud.com.json]:
```suggestion
    "documentation": "https://support.sendcloud.com/hc/en-gb/articles/4418777976212#1",
```

File: entries/s/sendcloud.com.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "Example Site": {
+    "domain": "sendcloud.com",
+    "url": "https://account.sendcloud.com",
+    "img": "sendcloud.com.png",
```
Comment (Carlgo11) [inline entries/s/sendcloud.com.json]: SVG files are proffered over PNG files. Here's one that you can use instead:
svg-icon
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks for the pull request!

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**